### ISSUE TITLE
Source of the issue regarding /ct simulate and it's ability to trigger Triggers

### ISSUE THREAD

**Issue by mattco98** [ChatTriggers/ChatTriggers#18]
When doing `/ct simulate [msg]`, it is unlikely to actually trigger any of the user's trigger because of the way it creates the `ClientChatReceivedEvent`. I know the exact reason, but cannot figure out a solution.

The issue lies in this line; specifically in regards to the ChatComponentText instantiation: 
```java
ClientChatReceivedEvent event = new ClientChatReceivedEvent((byte) 0, new ChatComponentText(toSend.toString()));
```

`toSend.toString()` is completely accurate. However, when logging `event.message.getFormattedText()`, there is suddenly a ` §r` at the end. That extra text is the reason for the issue - it prevents some trigger criteria from matching the command arguments.

It would be an easy fix if it was this way for every `ClientChatReceivedEvent`, however the part I can't figure out is that incoming chat events do not have this extra formatting at the end. They must somehow be created differently, and I can't quite find out how. All I found was the forge code that results in the ` §r`:
```java
/**
 * Gets the text of this component, with formatting codes added for rendering.
 */
public final String getFormattedText() {
    StringBuilder stringbuilder = new StringBuilder();

    for (IChatComponent ichatcomponent : this) {
        stringbuilder.append(ichatcomponent.getChatStyle().getFormattingCode());
        stringbuilder.append(ichatcomponent.getUnformattedTextForChat());
        stringbuilder.append((Object)EnumChatFormatting.RESET);
    }

    return stringbuilder.toString();
}
```

Basically,
- Calling `event.message.getFormattedText()` on an incoming `ClientChatReceivedEvent` from forge produces `<the text>`
- Calling `event.message.getFormattedText()` on a `ClientChatReceivedEvent` that was created manually produces `<the text> §r`

**Comment by mattco98** [comment]
nvm I'm just bad
